doc/modules/language-guide/pages/upgrades.adoc

```diff
@@ -1,14 +1,16 @@
 = Stable variables and upgrade methods
 
-One key feature of the {IC} is the ability to persist application data using WebAssembly memory rather than a traditional database. 
-This data storage model—referred to as **orthogonal persistence**—requires the ability to preserve and migrate the application state to support application upgrades. 
-For example, if you want to deploy a new version of an application to fix an issue or add functionality, you need to ensure that any application state that needs to be preserved remains unchanged by the upgrade process.
-The orthogonal persistence model allows an application deployed as a canister to save and restore
-state from dedicated _stable_ storage that is retained across an upgrade.
+One key feature of the {IC} is its ability to persist canister smart contract state state using WebAssembly memory and globals rather than a traditional database. This means that that the entire state of a canister is transparently restored before and saved after each message, without explicit user instruction. This automatic and user-transparent preservation of state is called _orthogonal persistence_.
```

Comment:
```suggestion
One key feature of the {IC} is its ability to persist canister smart contract state using WebAssembly memory and globals rather than a traditional database. This means that that the entire state of a canister is magically restored before, and saved after, each message, without explicit user instruction. This automatic and user-transparent preservation of state is called _orthogonal persistence_.
```